### PR TITLE
Support `TO` in  `CREATE VIEW` clause for Clickhouse

### PR DESCRIPTION
**Pull request by Bidaya0**
https://github.com/sqlparser-rs/sqlparser-rs/issues/1312

### CODE REVIEW

**Comment by Bidaya0** [comment]
@iffyio  
Thanks , I have made the changes as you suggested.

File: src/ast/mod.rs [hunk 1]
```diff
@@ -2029,6 +2029,7 @@ pub enum Statement {
         if_not_exists: bool,
         /// if true, has SQLite `TEMP` or `TEMPORARY` clause <https://www.sqlite.org/lang_createview.html>
         temporary: bool,
+        to: Option<ObjectName>,
```
Comment (iffyio) [inline src/ast/mod.rs:2032]: Could we add the description and a link to the docs where the operation comes from?

**Review by iffyio — COMMENTED** [review]
left a minor comment regarding docs, otherwise LGTM! cc @alamb

File: src/ast/mod.rs (hunk 1 above)
Comment (Bidaya0) [reply src/ast/mod.rs:2034]: Okay, I added a brief description and a link to Clickhouse views

**Review by alamb — APPROVED** [review]
Thank you @Bidaya0  and @iffyio  for the erview

**Comment by alamb** [comment]
I pushed a change to this PR to resolve `cargo fmt`

**Pull request merged**